### PR TITLE
[leaf2mqtt] Migrate to common v3

### PR DESCRIPTION
**Pull request by mitsumaui**
**Description of the change**

Update leaf2mqtt to use new common chart 3.1.0

**Benefits**

uses new common chart

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

File: charts/stable/leaf2mqtt/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ maintainers:
 dependencies:
 - name: common
   repository: https://library-charts.k8s-at-home.com
-  version: 2.5.0
+  version: 3.1.0
```
Comment (bjw-s) [inline charts/stable/leaf2mqtt/Chart.yaml:24]:
```suggestion
  version: 3.1.1
```

File: charts/stable/leaf2mqtt/values.yaml [hunk 2]
```diff
@@ -32,9 +32,6 @@ env: {}
   # CHARGING_UPDATE_INTERVAL_MINUTES:
   # LOG_LEVEL:
 
-service:
```
Comment (bjw-s) [inline charts/stable/leaf2mqtt/values.yaml:35]: Is there a UI/similar service that needs to be exposed? If so, a port number will need to be specified.

**Review by bjw-s — CHANGES_REQUESTED** [review]
Thanks for creating this PR 👍! Before we approve the PR, could you please update the chart's documentation (changelog and helm-doc strings, and regenerate the README) as well?

File: charts/stable/leaf2mqtt/Chart.yaml (hunk 1 above)
Comment (mitsumaui) [reply charts/stable/leaf2mqtt/Chart.yaml:24]: Ah so we prefer to follow the common chart version where possible? That's fine

File: charts/stable/leaf2mqtt/values.yaml (hunk 2 above)
Comment (mitsumaui) [reply charts/stable/leaf2mqtt/values.yaml:35]: No Service / listeners, only outbound communication.

**Comment by mitsumaui** [comment]
> Thanks for creating this PR +1! Before we approve the PR, could you please update the chart's documentation (changelog and helm-doc strings, and regenerate the README) as well?

Now - I thought there was somewhere this happened automagically - or was in a task? Is there something I'm missing on how to manage / update the docs as failing to find it in the kah chart docs?

File: charts/stable/leaf2mqtt/Chart.yaml (hunk 1 above)
Comment (mitsumaui) [reply charts/stable/leaf2mqtt/Chart.yaml:24]: Ah - ignore this going crazy :)

**Review by bjw-s — CHANGES_REQUESTED** [review]

**Comment by bjw-s** [comment]
I've gone ahead and migrated the chart in #1077. Still thanks a lot for the time you put in to this PR!

**Comment by mitsumaui** [comment]
> I've gone ahead and migrated the chart in #1077. Still thanks a lot for the time you put in to this PR!

Thanks for sorting that - apologies I didn't finish this out!

**Pull request closed without merging**